docs/ivys.js

```diff
@@ -31,6 +33,23 @@ IvyApp.prototype.addItem = function (imageFileName) {
 IvyApp.prototype.removeItem = function () {
     // removes an image from masonry grid
     "use strict";
+    
+};
+
+IvyApp.prototype.addTag = function (tagName) {
+    // Add a tag to the page
+    "use strict";
+    var tagElement = '<li><a class="tag" href="#">' +
+        tagName +
+        '<i class="fa fa-times" aria-hidden="true"></i> \
+        </a></li>';
+    this.$tagList.append(tagElement);
+};
+
+IvyApp.prototype.removeTag = function () {
+    // Remove a tag from the page
+    "use strict";
+    // Should I remove by tag name?
```

Comment:
At any rate, if the element is something you need to be able to search for and reliably find, it should probably have an id.